src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -215,6 +218,14 @@ class ShenandoahHeap : public CollectedHeap {
 private:
            size_t _initial_size;
            size_t _minimum_size;
+           size_t _promotion_potential;
+           size_t _promotion_in_place_potential;
+           size_t _pad_for_promote_in_place;    // bytes of filler
+           size_t _promotable_humongous_regions;
+           size_t _promotable_humongous_usage;
+           size_t _regular_regions_promoted_in_place;
+           size_t _regular_usage_promoted_in_place;
+
```

Comment:
Although these names may in many cases be "self-explanatory" to those familiar with the code, I think they need at least a brief 1-line description each.